### ISSUE TITLE
The performance of this baseline project

### ISSUE THREAD

**Issue by Aimtrueflw** [knowledgedefinednetworking/RouteNet-challenge#7]
I have tried to use this baseline project, but I have doubts about the performance of this baseline. Without changing the training parameters, the final MAPE is not satisfactory. And you did not announce the normal MAPE level of this baseline. Is it necessary to modify the model to further improve performance, or is it the problem that my local training did not converge?

Epoch 100/100
1000/1000 [==============================] - 193s 193ms/step - loss: 0.0218 - mean_absolute_percentage_error: 68.2392 - val_loss: 0.0286 - val_mean_absolute_percentage_error: 52.7838

**Comment by jsuarezv** [comment]
Hello,
This model was presented as a baseline for the Graph Neural Networking challenge ([https://bnn.upc.edu/challenge/](https://bnn.upc.edu/challenge/)). It is not expected to achieve a good MAPE, as it just represents a starting point for participants.
If you are interested in achieving a good MAPE, you may refer to the winning solutions of the challenge:
- Graph Neural Networking challenge 2020 [[link](https://bnn.upc.edu/challenge/gnnet2020/)]:
Dataset: [https://bnn.upc.edu/challenge/gnnet2020/ch20-dataset/](https://bnn.upc.edu/challenge/gnnet2020/ch20-dataset/)
2nd best solution (the 1st one was not open sourced) -> [https://github.com/ITU-AI-ML-in-5G-Challenge/PS-014.2_GNN_Challenge_SalzburgResearch](https://github.com/ITU-AI-ML-in-5G-Challenge/PS-014.2_GNN_Challenge_SalzburgResearch)
- Graph Neural Networking challenge 2021 [[link](https://bnn.upc.edu/challenge/gnnet2021/)]:
Dataset: [https://bnn.upc.edu/challenge/gnnet2021/dataset/](https://bnn.upc.edu/challenge/gnnet2021/dataset/)
Winning solution (1st) -> [https://github.com/ITU-AI-ML-in-5G-Challenge/ITU-ML5G-PS-001-PARANA](https://github.com/ITU-AI-ML-in-5G-Challenge/ITU-ML5G-PS-001-PARANA)
